backend/src/card_game/core.clj

```diff
@@ -9,18 +9,16 @@
 
 (defn new-player
   "Creates a new player object"
-  ([] (new-player 12))
-  ([cards]
+  ([]
```

Comment:
Did we lose the ability to create players with different starting hands? (not that it was ever an extremely useful ability or one that we actually needed)

Reply:
We were not able to do so without making several other changes.
Since, right now, hands are defined on `configs/ini-hand` I see no reason to have that.
At some point we will implement different hands and then will be the time to work this out. Having a structure like the one we had doesn't help either for that.